Update _sidebar.md (#146)
Suggester/Documentation: topics/Donation-Custom-Instances.md

> # Donations / Custom Instances

Reviewer: ```suggestion
# Donations & Custom Instances
```

---
### Donations

> Hello!

Reviewer: ```suggestion
```

> If you are looking for a way to support and help keep the costs down for Suggester, you can do a one time donation or purchase a tier off our donation page [here](https://ko-fi.com/suggester).

Reviewer: ```suggestion
If you are looking for a way to support and help keep the costs down for Suggester, you can do a one-time donation or purchase a tier off our [donation page](https://ko-fi.com/suggester)!
```

> Our one time donators and tier buyers are able to get a fancy purple donor role in our [Discord Support Server.](https://suggester.js.org/support)

Reviewer: ```suggestion
Our one time donators and tier buyers are able to get a fancy purple donor role in the [support server](https://suggester.js.org/support).
```

### Custom Instances

> If you are looking for a way to support our bot and to be able to get your own personally branded custom instance of Suggester, we have several tiers.

Reviewer: ```suggestion
If you are looking for a way to support our bot and get your own custom instance of Suggester, we have several tiers available!
```

| Tiers                                                                              |              Rewards              |       Price        |
|------------------------------------------------------------------------------------|:--------------------------------------------:|----------------------|

> | [Tier One](https://ko-fi.com/summary/54a88943-f7f1-4c2d-8ed8-076d616aeff8)         | A Donor role in the [support server](https://suggester.js.org/support) and a Custom Instance for one server. | $5 per month |

Reviewer: ```suggestion
| [Silver Tier](https://ko-fi.com/summary/54a88943-f7f1-4c2d-8ed8-076d616aeff8)         | A Donor role in the [support server](https://suggester.js.org/support) and a Custom Instance for one server. | $5 per month |
```

> | [Tier Two](https://ko-fi.com/summary/d5964004-f812-4b15-9c1a-4f8718787217)         | A Donor role in the [support server](https://suggester.js.org/support) and a Custom Instance for two servers. | $8 per month |

Reviewer: ```suggestion
| [Gold Tier](https://ko-fi.com/summary/d5964004-f812-4b15-9c1a-4f8718787217)         | A Donor role in the [support server](https://suggester.js.org/support) and a Custom Instance for two servers. | $8 per month |
```

> | [Tier Three](https://ko-fi.com/summary/415f47b3-26e7-4dd6-9b25-7b75fb8ac682)       | A Donor role in the [support server](https://suggester.js.org/support) and a Custom Instance for three servers. | $10 per month |

Reviewer: ```suggestion
| [Platinium Tier](https://ko-fi.com/summary/415f47b3-26e7-4dd6-9b25-7b75fb8ac682)       | A Donor role in the [support server](https://suggester.js.org/support) and a Custom Instance for three servers. | $10 per month |
```
